config/fcos/v1_6_exp/validate_test.go

```diff
@@ -231,3 +231,129 @@ func TestValidateGrubUser(t *testing.T) {
 		})
 	}
 }
+
+func TestValidateConfig(t *testing.T) {
+	tests := []struct {
+		config  Config
+		err     error
+		errPath path.ContextPath
+	}{
+		// valid config
+		{
+			config: Config{
```

Comment:
If you take a closer look to the test's above, it might make sense to follow a similar format. 

_Hint: look at how the elements in the array are initialized._